### ISSUE TITLE
Simply rerelease v2.6.4 as v2.6.6

### ISSUE THREAD

**Issue by maconfr** [jakubroztocil/rrule#417]
For users of the package, the current situation is unfortunate. A simple workaround, which is more convenient for the users of this package, would be to republish the working version as v2.6.6.

Maybe it is also possible to mark v2.6.4 as "latest" at npm or delete the broken version?

**Comment by Woodz** [comment]
You can not delete a broken version. npm packages are cached throughout the Internet, including on developers machines and build agents. The correct approach to a broken version is to release a new fixed version (either fixing the bug or rolling the changes back). A good article to read is https://doughellmann.com/blog/2016/02/25/so-youve-released-a-broken-package-to-pypi-what-do-you-do-now/.

**Comment by konrad-garus** [comment]
Yes please!

**Comment by Jogai** [comment]
Yes, like I said before: https://github.com/jakubroztocil/rrule/issues/416#issuecomment-679013893

**Comment by abbathaw** [comment]
Any idea when this will be fixed?

**Comment by Jaimeloeuf** [comment]
Just tagging the key contributors, if anyone sees it to help to release a new version on npm.
Since the roll back also means that the issues previously fixed by v2.6.5 are now still present in v2.6.4.

It would be great to have a new version to fix all these and the underlying issue of 2.6.5 else, it would be great to have v2.6.4 as the latest version first, since tools like <https://www.npmjs.com/package/npm-check-updates> will still use v2.6.5 even if it is marked as deprecated!

@davidgoli @jakubroztocil @david-golightly-leapyear @arolson101

**Comment by jakubroztocil** [comment]
I’m sorry about about the broken release. I thought marking it as deleted on npm would have a similar effect like [yanking](https://www.python.org/dev/peps/pep-0592/) a Python package. I’ve just released 2.6.6 that should be good.
